pages/common/xmllint.md

```diff
@@ -0,0 +1,32 @@
+# xmllint
+
+> XML parser.
+> Also useful for detecting errors in XML code.
```

Comment:
I think this can be combined in the first line itself by saying "XML parser and linter."